jetty-demos/jetty-servlet4-demos/pom.xml

```diff
@@ -32,6 +32,53 @@
     <warSourceDirectory>${project.build.directory}/webapp</warSourceDirectory>
   </properties>
 
+  <dependencyManagement>
```

Comment:
This is precisely what I was trying to avoid - the duplication of dependency versions between the `jetty-eeX` poms and the `jetty-demos`.

Reply:
Read the issue text

Reply:
The jetty-demos don't depend on Jetty EE, they depend on Jakarta EE.
It was wrong to try to shoehorn them into Jetty EE.

Reply:
> Read the issue text

I have.

> The jetty-demos don't depend on Jetty EE, they depend on Jakarta EE. It was wrong to try to shoehorn them into Jetty EE.

The `jetty-demos` don't depend on any `jetty-eeX` module, they just depend on some of the _same dependencies_ of those `jetty-eeX` modules. 


> The jetty-demos don't depend on Jetty EE, they depend on Jakarta EE. It was wrong to try to shoehorn them into Jetty EE.

I don't understand your argument here.  There is no relationship between any `jetty-eeX` module and the `jetty-demos` modules.  The `jetty-eeX` modules depend on Jakarta EE and _so do the_ `jetty-demos`:

Servlet 4 `javax.servlet api`  is depended on by:  `jetty-ee8`  and `jetty-servlet4-demos`

Servlet 5 `jakarta.servlet api` is depended on by: `jetty-ee9`  and `jetty-servlet5-demos`

Servlet 6 `jakarta.servlet api` is depended on by: `jetty-ee10`

Servlet 6.1 `jakarta.servlet.api` is depended on by: `jetty-ee11`

If you want to declare the various servlet spec dependencies outside of the top level pom, say in  `servlet4/pom.xml`, `servlet5/pom.xml`, `servlet6/pom.xml`, `servlet6.1/pom.xml` and then have `jetty-ee8` and `jetty-servlet4-demos` depend on `servlet4/pom.xml`, `jetty-ee9` and `jetty-servlet5-demos` depend on `servlet5/pom.xml` ,  etc etc then fine. That keeps the version information in a single place. My objection is to duplicating the version information across the project.